src/components/problemDetailsModal/problemDetailsModal.tsx

```diff
@@ -44,34 +33,37 @@ const ProblemDetailsModalBody = (props: { problem: DetailsType | null }) => {
   return (
     <>
       <Flex>
-        <Badge colorScheme="red">ERROR {problem?.status}</Badge>
+        <Badge colorPalette="errorPalette">ERROR {problem?.status}</Badge>
       </Flex>
       <Flex my="20px">
-        <Card>
-          <Code colorScheme="grey">{problem?.originalDetail ? problem.originalDetail : problem?.message}</Code>
-        </Card>
+        <Card.Root>
+          <Card.Body>
+            <Code colorPalette="greyPalette">
```

Comment:
consider creating a 'codeBlockPalette' in theme so that can be customized based on the branded theme.